client/scss/styleguide/_styleguide.scss

```diff
@@ -25,30 +22,72 @@
   }
 }
 
-.styleguide__hex {
+.styleguide__palette__heading {
+  @include font('HNM4');
+  text-transform: uppercase;
+  flex-basis: 100%;
+  margin-bottom: 1em;
+}
+
+.styleguide__palette__section {
+  display: flex;
+  flex-wrap: wrap;
+  border-bottom: 3px solid color('cream');
+  padding-bottom: 15px;
+  margin-bottom: 30px;
+
+  &:last-child {
+    border: 0;
+  }
+}
+
+.styleguide__palette__block {
+  flex-basis: 25%;
+  margin-bottom: 15px;
+}
+
+.styleguide__palette__color {
   position: relative;
-  width: 200px;
-  height: 200px;
+  min-width: 200px;
+  margin-right: 15px;
+  border: 1px solid color('transparent');
+
+  &--white {
+    border-color: color('accessible-grey-2');
+  }
+
+  &:before {
+    content: '';
+    display: block;
+    padding-top: 100%;
+  }
 }
 
-.styleguide__color {
-  position: absolute;
-  top: 5px;
-  left: 5px;
-  background: color('white');
-  padding: 5px;
-  max-width: 190px;
-  border-radius: 3px;
-  border: 1px solid color('smoke');
+.styleguide__palette__hyphen {
+  visibility: hidden;
+}
+
+.styleguide__palette__name {
```

Comment:
Not sure why we need this namespace... BEM isn't really meant for name-spacing.
To namespace we could use the more traditional SASS:
```SASS
.styleguide {
  .palette__hyphen {}
}
```

which I suppose denotes that we only use it in styleguide, but I'm not sure where the value in that information lies?